sources/packages/web/src/components/layouts/Institution/sidebar/HomeSideBar.vue

```diff
@@ -120,6 +120,18 @@ export default defineComponent({
                   },
                 },
               },
+              {
+                title: "Request a Change",
```

Comment:
Question here to @hellolynn-tbtb 

Is it ok to have a button called `Request a Change` for student(request an appeal) and in institutions for different purpose?

![image](https://github.com/bcgov/SIMS/assets/54600590/4ee8a335-0904-4ba7-91c6-abc49bdffedd)

Reply:
@dheepak-aot @ann-aot hey yup - business is okay with it. From UX - I don't see it being a confusing experience as it is within their own portal. Thanks